src/components/ButtonSubmit.js

```diff
@@ -13,6 +13,7 @@ export const ButtonSubmit = (props) => {
       accessibilityLabel={props.title}
       accessibilityHint={props.fieldHint}
       disabled={props.disabled}
+      loading={props.loading}
```

Comment:
I decided to add this so we can see when the button click has been registered clearly to be able to test the performance issue better as have been unable to replicate on the emulators, will test tomorrow in the office